pkg/manifests/config.go

```diff
@@ -29,6 +29,11 @@ import (
 
 const (
 	DefaultRetentionValue = "15d"
+
+	// Limit the body size from scrape queries
+	// Assumptions: one node has maximum 110 pods, each pod exposes 100 metrics, each metric is expressed by at most 100 bytes.
+	// 1.5x the size for a safe margin, it rounds up to 2MB.
+	DefaultScrapeBodySizeLimit = "2MB"
```

Comment:
Do we want to allow users to set this value themselves instead of us setting an arbitrary value?
Looking at the metrics coming from kube-state-metrics in the daily cluster, the 2MB are can very easily be exceeded

![image](https://user-images.githubusercontent.com/1286231/143391315-f73a1476-d30a-4df5-82b0-40db2e893b15.png)

Reply:
I agree to a certain extent, we can't limit the body size for UWM, however we can judge the same for PM.

Reply:
I'm not sure if we can even make an estimate for PM because it depends on the size of the cluster. We have basic clusters with just a few pods, and also build clusters with ~10K pods. Exporters like KSM and NE will produce data with different size depending on the size of the cluster, so I'm not sure how we could cap this properly.

Reply:
We can let the user to choose a size they see fit. How about `limitScrapeBodySize `be a string option instead of boolean? and the user can set it to the following values:
1. a size in Prometheus config format such as "10MB"
2. a string "default" to use the default value we provided "2MB"
3. empty. no limit will be set.

Reply:
I personally would have only 1. and 3., but I also don't know what the actual requirements are. Maybe @simonpasquier could chime in.